skriptpy/core.py

```
@@ -0,0 +1,620 @@
+from contextlib import contextmanager
+from typing import List, Callable
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)